src/plenigo/internal/services/Service.php

```diff
@@ -74,6 +76,8 @@ protected static function getRequest($endPoint, $oauth = false, array $params =
     protected static function postRequest($endPoint, $oauth = false, array $params = array())
     {
         if($oauth){
```

Comment:
This was discussed with Rob, creating a method for a single condition because 3 lines of code are repeated would decrease readability and performance  improvement is negligible.